lib/open_project/translations/models/locales_updater.rb

```diff
@@ -0,0 +1,127 @@
+require 'pathname'
+require 'fileutils'
+
+require_relative '../helpers/tmp_directory'
+require_relative './git_repository'
+require_relative './i18n_provider'
+require_relative './locales_updater_configuration'
+
+ENGLISH_TRANSLATION_FILE = 'en.yml'
+ENGLISH_JS_TRANSLATION_FILE = 'js-en.yml'
+ACCEPTANCE_LEVEL = ENV['ACCEPTANCE_LEVEL'].nil? ? 100: ENV['ACCEPTANCE_LEVEL'].to_i
```

Comment:
While I get that this should mimic a constant (which do not really exist in ruby), this should probably be a method, as there is logic associated with it:

```ruby
def acceptance_level
  return 100 if ENV['ACCEPTANCE_LEVEL'].nil?
  ENV['ACCEPTANCE_LEVEL'].to_i
end
```

Also - and this is just bitching - when I set the `ENV['ACCEPTANCE_LEVEL'] to something like `3e`, the level will be `0`.

At elast there is a space missing, if you want to keep it that way :wink: